docs/content/tutorials/monitoring.md

```diff
@@ -4,4 +4,67 @@ title: Monitoring
 
 # Monitoring
 
-TBD
+Monitoring your forward proxy is essential for ensuring its health, performance, and resource utilization.
+This document provides a step-by-step guide on how to enable the API server, collect Prometheus metrics, and set up visualization tool.
+
+## Installation
+
+### Enabling API Server 
+
+To enable the API server and expose Prometheus metrics, use the `--api-address` flag.
+
+```bash
+  forwarder run --api-address :10000
+```
+
+### Accessing Metrics 
+
+Once the API server is running, you can access Prometheus metrics using tools like curl. 
+
+```bash 
+  curl localhost:10000/metrics
+```
+
+Adjust the URL based on the address you specified with the `--api-address` flag.
+
+### Configure Prometheus 
+
+To configure Prometheus to scrape metrics from your Forwarder, follow these steps:
+- Open your Prometheus configuration file (usually `prometheus.yml`).
+- Add a new job to the `scrape_configs` section:
+
+```yaml
+scrape_configs:
+- job_name: 'forwarder'
+  scrape_interval: 15s
+  static_configs:
+    - targets: ['forwarder-api-server-address']
+```
+
+- Replace `forwarder-api-server-address` with the actual address where your Forwarder's API server is running.
+Save the changes and restart Prometheus.
+- Adjust the Prometheus configuration based on your requirements.
+
+### Configure Grafana
+1. Install Grafana and start the Grafana server.
+1. Access the Grafana web interface.
+1. Add Prometheus as a data source:
+   - Go to `Settings` > `Data Sources.`
+   - Click `Add your first data source.`
+   - Choose Prometheus and configure the URL (e.g., `http://localhost:9090`).
+1. Import Dashboard from Grafana Registry:
+   - Go to the Grafana Dashboard.
+   - Click on the `New` or `+` icon select `Import.`
+   - Copy Dashboard [source code](https://github.com/saucelabs/forwarder/blob/main/local/monitoring/dashboards/forwarder.json) from github.
+1. Configure the Data Source:
+   - Set the data source variable to the Prometheus one you added earlier.
+1. Run Dashboard with the Right Namespace:
+   - Set the `prom_namespace` variable to the value configured in your Forwarder with the `--prom-namespace` flag.
+
+Congratulations! You have successfully set up monitoring for your Forwarder using Prometheus and Grafana. Adjust the configuration parameters as needed based on your specific setup.
+
+## Try our local monitoring setup
+
+We have provided a local monitoring setup that you can use to try out the monitoring features of the Forwarder.
+All you need is a Docker to run Prometheus, Grafana and Forwarder.
+Visit [here](https://github.com/saucelabs/forwarder/blob/main/local/monitoring/README.md) for more info. 
```

Comment:
See [local/monitoring](https://github.com/saucelabs/forwarder/blob/main/local/monitoring) for more details.